app/controllers/contents_controller.rb

```diff
@@ -28,9 +28,13 @@ def show_table
   def update
     authorize! @content
 
-    update_files
-
-    head :ok
+    if valid?
+      update_files
+      head :ok
+    else
+      render json: { error: 'Unable to upload file due to: Duplicate file. Delete the file and try again.' },
```

Comment:
Multiple files will be uploaded at a time -- this doesn't indicate which file is the dupe.